cli/src/config/colors.toml

```diff
@@ -79,7 +79,9 @@
 "diff file_header" = { bold = true }
 "diff hunk_header" = "cyan"
 "diff removed" = "red"
+"diff removed whitespace" = { bg = "red" }
```

Comment:
You could change the `fg` to black for debugging (perhaps leave a comment). Currently, your test looks like this in `less`; it took me a while to figure out what happened.

![image](https://github.com/martinvonz/jj/assets/4123047/a2e26c02-e9f3-4113-a232-35dc177546cc)